internal/fileutils/fileutils_test.go

```diff
@@ -0,0 +1,144 @@
+package fileutils_test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/ubuntu/authd-oidc-brokers/internal/fileutils"
+)
+
+func TestFileExists(t *testing.T) {
+	tests := map[string]struct {
```

Comment:
We can make this parallel, no?

I won't comment on every single instance, so please take a look at the other tests also :wink:

Reply:
good catch, fixed